hack/verify-openapi-spec.sh

```diff
@@ -32,14 +32,15 @@ TMP_SPECROOT="${KUBE_ROOT}/_tmp/openapi-spec"
 _tmp="${KUBE_ROOT}/_tmp"
 
 mkdir -p "${_tmp}"
-trap "rm -rf ${_tmp}" EXIT SIGINT
 cp -a "${SPECROOT}" "${TMP_SPECROOT}"
+trap "cp -a ${TMP_SPECROOT} \"${KUBE_ROOT}/api\"; rm -rf ${_tmp}" EXIT SIGINT
```

Comment:
Yes, it is works and better. I just (blindly) copy/paster the command on (old) line 42. Would use your first suggestion.